pkg/util/utils_test.go

```diff
@@ -2,9 +2,11 @@ package util
 
 import (
 	"fmt"
+	"path/filepath"
 	"testing"
 	"time"
 
+	"github.com/docker/docker/pkg/homedir"
```

Comment:
Why not containers/storagek/pkg/homedir

Reply:
+1, this test does not test the correct package/code change

Reply:
Wow. It looks like I selected the wrong path when using auto import in `Goland` 😆. It has now been corrected ✅.